games-board/megamek/megamek-0.34.10-r2.ebuild

```diff
@@ -1,21 +1,23 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit eutils java-pkg-2 java-ant-2
+EAPI=7
+inherit desktop edos2unix java-pkg-2 java-ant-2
 
-DESCRIPTION="an unofficial, online version of the Classic BattleTech board game"
+JAVA_PKG_WANT_SOURCE=1.5
+JAVA_PKG_WANT_TARGET=1.5
```

Comment:
@fordfrog I was unable to figure out a way to get this package to compile without these variables. But the documentation specifically states that these should only be used for testing.

What other options do I have?

Reply:
those are really not needed. you specify the source and target by `DEPEND` (source) and `RDEPEND` (target), at least i think it's this way.

Reply:
I took them out, thanks for the help.